pkg/i18n/french.go

```diff
@@ -76,30 +76,30 @@ func frenchSet() TranslationSet {
 		StandaloneContainersTitle: "Conteneurs Autonomes",
```

Comment:
```suggestion
		StandaloneContainersTitle: "Conteneurs autonomes",
```
Pour tous les titres, je serais partisan de ne mettre de majuscule qu'au premier mot, comme il est généralement d'usage en français.

![image](https://github.com/jesseduffield/lazydocker/assets/25877142/8f80c831-e558-4693-a134-0578876c5fba)